index.html

```diff
@@ -8,10 +8,21 @@
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
   <link rel="stylesheet" type="text/css" href="application.css">
+  <link rel='stylesheet' type='text/css' href='http://fonts.googleapis.com/css?family=Roboto'>
 </head>
 
 <body>
   <h1 class="title">Front-End Tools</h1>
+  <div name="tech" class="m-left-10">
+    <div class="tech-border">
+      <div class="tech-border tech-content">
+        <img src="assets/react_icon.svg" class="tech-logo" alt="A React logo">
+        <h4 class="tech-name">ReactJS</h4>
```

Comment:
h3 is "more important" than h4 and since the tech name is more important than the tech release, this should be an h3 and the tech release an h4